metadata.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "puppetlabs-mount_core",
+  "version": "0.1.0",
+  "author": "Puppet Labs",
+  "summary": "",
```

Comment:
Planning on setting summary, site, issues, etc. later when going through for the 1.0.0 release?